parsons/action_kit/action_kit.py

```diff
@@ -186,28 +185,12 @@ def get_events(self, limit=None, **kwargs):
 
                 .. code-block:: python
 
-                    ak.get_events(fields__name__contains="FirstName")
+                    ak.get_events(name__contains="FirstName")
```

Comment:
Is this backwards-incompatible or was the doc just incorrect?

Reply:
I think this is fixing the documentation but not sure. Tagging @sjwmoveon since she's the author of this change.

Reply:
Yes, the documentation was incorrect.

Reply:
okay, great. I'm going to go ahead and approve + merge this. thanks for the contribution @sjwmoveon and @crayolakat and thanks for the review @schuyler1d